iothub/service/samples/readme.md

```diff
@@ -1,14 +1,85 @@
-Service samples were moved to [Azure-Samples/azure-iot-samples-csharp][samples-repo] repository
+## Service samples for Azure IoT Service SDK for C#
 
-### [Service samples][service-samples]
-* [ADMSample][adm-sample]
-* [JobsSample][jobs-sample]
-* [RegistryManagerSample][reg-man-sample]
-* [ServiceClientSample][service-client-sample]
+Here, you'll find simple samples showing how to use the various features of the Microsoft Azure IoT Hub service from a back-end application running C# code.
+
+### List of Samples
+
+* [ADM Sample][adm-sample]
+* [Jobs Sample][jobs-sample]
+* [Registry ManagerSample][reg-man-sample]
+* [Service Client Sample][service-client-sample]
+* [Plug and play Samples ][pnp-service-sample]
+
+### Prerequisites
+In order to run the device samples on Linux or Windows, you will first need the following prerequisites:
+* [Setup your IoT hub][lnk-setup-iot-hub]
+* After creating the IoT Hub, retreive the iothubowner connection string (mentioned in instructions to create the hub).
+
+### Setup environment
+.NET Core SDK 3.0.0 or greater on your development machine. You can download the .NET Core SDK for multiple platforms from .NET. You can verify the current version of C# on your development machine using 'dotnet --version'.
```

Comment:
We support .NET Standard 2.0, so ideally, you can run it with a .NET Core 2.0 application as well